react/components/EditorContainer/StoreEditor/Styles/StyleEditor/mutations/DeleteFontFamily.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
```

Comment:
@alcar brought up the point that this file doesn't use JSX, so it should be renamed to `.ts` instead of `.tsx`.